src/Basemaps/StamenTonerDark.js

```diff
@@ -5,6 +5,7 @@ import olSourceStamen from 'ol/source/stamen'
 import { BasemapOption, BasemapThumbnail, Label } from './styled'
 import { stamenTonerDark } from './thumbnails'
 import { connectToMap } from 'Map'; // eslint-disable-line
+import translations from 'locales/en'
```

Comment:
`translations` should be supplied by `connectToMap` as a prop and it's defaulted to `locales/en` in that `<Map>` file to have a single source of truth (instead of importing translations into every component file--which may cause bugs if someone is actually trying to override translations on the `<Map>` level and each is also defaulting translations)